From: Dmitry Timoshkov <dmitry@baikal.ru>
Subject: [PATCH 1/3] comctl32/tests: Print id value as decimal. (Resend)
Message-Id: <20220125115654.f6c546b8970023e4daeb014c@baikal.ru>
Date: Tue, 25 Jan 2022 11:56:54 +0300

All the notification codes (printed as an id) have negative base value
(SBN_, UDN_, HDN_, BCN_, TBN_, TTN_, RBN_, TRBN_, PGN_, TVN_, LVN_, TCN_,
CBEN_, IPN_, MCN_, DTN_), and printing them as decimal helps to quickly
identify the corresponding notification value in cases when an id in
the message sequence doesn't match. Custom ids in the message sequences
have small numbers (0, 1, 2, ...), so they also get printed correctly.

Signed-off-by: Dmitry Timoshkov <dmitry@baikal.ru>
---
 dlls/comctl32/tests/msg.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dlls/comctl32/tests/msg.h b/dlls/comctl32/tests/msg.h
index 11111111111..11111111111 100644
--- a/dlls/comctl32/tests/msg.h
+++ b/dlls/comctl32/tests/msg.h
@@ -261,14 +261,14 @@ static void ok_sequence_(struct msg_sequence **seq, int sequence_index,
                         failcount++;
                         dump++;
                         ok_(file, line) (FALSE,
-                            "%s: in msg 0x%04x expecting id 0x%x got 0x%x\n",
+                            "%s: in msg 0x%04x expecting id %d got %d\n",
                             context, expected->message, expected->id, actual->id);
                     }
                 }
                 else
                 {
                     ok_(file, line) (expected->id == actual->id,
-                        "%s: in msg 0x%04x expecting id 0x%x got 0x%x\n",
+                        "%s: in msg 0x%04x expecting id %d got %d\n",
                         context, expected->message, expected->id, actual->id);
                     if (expected->id != actual->id) dump++;
                 }

-- 
2.35.1

